generators/server/templates/src/main/java/package/web/rest/AccountResource.java.ejs

```diff
@@ -88,7 +86,7 @@ public class AccountResource {
     }
 }
 <%_ } else { _%>
-import com.codahale.metrics.annotation.Timed;
+import io.micrometer.core.annotation.Timed;
```

Comment:
I think the imported can be removed (as the methods are not annotated with @Timed anymore.